lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -524,120 +376,13 @@ def non_tract_locations(resource: dict):
-        location_references = non_tract_references,
-        diagnosis = diagnosis,
-        contained = contained
-    )
-
-    return create_entry_and_reference(encounter_resource, "Encounter")
-
-
-def create_resource_condition(record: dict, symptom_name: str, patient_reference: dict, onset_date:str) -> Optional[dict]:
-    """ Returns a FHIR Condition resource. """
-    mapped_symptom_name = map_symptom(symptom_name)
-    if not mapped_symptom_name:
-        return None
-
-    # XXX TODO: Define this as a TypedDict when we upgrade from Python 3.6 to
-    # 3.8.  Until then, there's no reasonable way to type this data structure
-    # better than Any.
-    #   -trs, 24 Oct 2019
-    condition: Any = {
-        "resourceType": "Condition",
-        "id": f'{mapped_symptom_name}',
-        "code": {
-            "coding": [
-                {
-                    "system": f"{INTERNAL_SYSTEM}/symptom",
-                    "code": mapped_symptom_name
-                }
-            ]
-        },
-        "subject": patient_reference
-    }
-
-    if onset_date:
-        condition["onsetDateTime"] = onset_date
-
-    return condition
-
-
-def create_specimen(record: dict, patient_reference: dict, collection_method: CollectionMethod) -> tuple:
-    """ Returns a FHIR Specimen resource entry and reference """
-
-    def specimen_barcode() -> Optional[str]:
-        """
-        Return specimen barcode from REDCap record.
-        """
-         # Normalize all barcode fields upfront.
-        barcode_fields = {
-            # Kiosk related
-            "collect_barcode_kiosk", # from	kiosk_registration_4c7f
-
-            # Swab and Send related
-            "pre_scan_barcode", # from test_fulfillment_form
-            "barcode_swabsend",	# from husky_test_kit_registration
-            "return_utm_barcode", # from post_collection_data_entry_qc
-        }
-
-        for barcode_field in barcode_fields:
-            record[barcode_field] = record[barcode_field].strip().lower()
-
-        return record["collect_barcode_kiosk"] or record["return_utm_barcode"] or \
-            record["pre_scan_barcode"] or None
-
-
-    barcode = specimen_barcode()
-
-    if not barcode:
-        LOG.warning("Could not create Specimen Resource due to lack of barcode.")
-        return None, None
-
-    specimen_identifier = create_identifier(
-        system = f"{INTERNAL_SYSTEM}/sample",
-        value = barcode
-    )
-
-    # YYYY-MM-DD HH:MM:SS in REDCap
-    received_time = record['samp_process_date'].split()[0] if record.get('samp_process_date') else None
-
-    note = None
-
-    if record['able_to_test'] == 'no':
-        note = 'never-tested'
     else:
-        note = 'can-test'
-
-    specimen_type = 'NSECR'  # Nasal swab.
-    specimen_resource = create_specimen_resource(
-        [specimen_identifier], patient_reference, specimen_type, received_time,
-        collection_date(record, collection_method), note
-    )
+        return None
```

Comment:
This else statement is unnecessary. In the next line, we'll return `encounter_date = None` if we don't enter the first two conditional blocks.

Reply:
Fixed in "FIXUP: Kairsten's feedback" commit.